aldryn_newsblog/templates/aldryn_newsblog/base.html

```diff
@@ -10,7 +10,7 @@
 
 {% block content %}
     <div class="aldryn aldryn-newsblog">
-        {% block blog_title %}<h2>{% trans "Blog" %}</h2>{% endblock %}
+        {% block blog_title %}<h2>{{ app_config.app_title }}</h2>{% endblock %}
```

Comment:
This can be written as {{ view.config.app_title }} without requiring any extra variable in context